data/cncf.yaml

```diff
@@ -1889,43 +1889,225 @@
+      url: https://github.com/open-telemetry/opentelemetry-java-contrib
+      check_sets:
+        - code-lite
+    - name: opentelemetry-java-examples
+      url: https://github.com/open-telemetry/opentelemetry-java-examples
+      check_sets:
+        - code
     - name: opentelemetry-java-instrumentation
       url: https://github.com/open-telemetry/opentelemetry-java-instrumentation
       check_sets:
         - code-lite
     - name: opentelemetry-js
       url: https://github.com/open-telemetry/opentelemetry-js
+      check_sets:
+        - code
+    - name: opentelemetry-js-contrib
+      url: https://github.com/open-telemetry/opentelemetry-js-contrib
       check_sets:
         - code-lite
-    - name: opentelemetry.io
-      url: https://github.com/open-telemetry/opentelemetry.io
+    - name: opentelemetry-lambda
+      url: https://github.com/open-telemetry/opentelemetry-lambda
       check_sets:
-        - docs
+        - code
+    - name: opentelemetry-network
+      url: https://github.com/open-telemetry/opentelemetry-network
+      check_sets:
+        - code
+    - name: opentelemetry-operator
+      url: https://github.com/open-telemetry/opentelemetry-operator
+      check_sets:
+        - code
+    - name: opentelemetry-php
+      url: https://github.com/open-telemetry/opentelemetry-php
+      check_sets:
+        - code
+    - name: opentelemetry-php-contrib
+      url: https://github.com/open-telemetry/opentelemetry-php-contrib
+      check_sets:
+        - code-lite
+    - name: opentelemetry-php-instrumentation
+      url: https://github.com/open-telemetry/opentelemetry-php-instrumentation
+      check_sets:
+        - code-lite
+    - name: opentelemetry-proto
+      url: https://github.com/open-telemetry/opentelemetry-proto
+      check_sets:
+        - code
+    - name: opentelemetry-proto-java
+      url: https://github.com/open-telemetry/opentelemetry-proto-java
+      check_sets:
+        - code
+    - name: opentelemetry-python
+      url: https://github.com/open-telemetry/opentelemetry-python
+      check_sets:
+        - code
+    - name: opentelemetry-python-contrib
+      url: https://github.com/open-telemetry/opentelemetry-python-contrib
+      check_sets:
+        - code-lite
+    - name: opentelemetry-ruby
+      url: https://github.com/open-telemetry/opentelemetry-ruby
+      check_sets:
+        - code
+    - name: opentelemetry-ruby-contrib
+      url: https://github.com/open-telemetry/opentelemetry-ruby-contrib
+      check_sets:
+        - code-lite
+    - name: opentelemetry-rust
+      url: https://github.com/open-telemetry/opentelemetry-rust
+      check_sets:
+        - code
+    - name: opentelemetry-rust-contrib
+      url: https://github.com/open-telemetry/opentelemetry-rust-contrib
+      check_sets:
+        - code-lite
+    - name: opentelemetry-sandbox-web-js
+      url: https://github.com/open-telemetry/opentelemetry-sandbox-web-js
+      check_sets:
+        - code
```

Comment:
my understanding is this is just a playground, and will eventually land in opentelemetry-js
```suggestion
```